src/Language/Fixpoint/Smt/Theories.hs

```diff
@@ -507,16 +470,26 @@ theorySymbols ds = fromListSEnv $  -- SHIFTLAM uninterpSymbols
 interpSymbols :: [(Symbol, TheorySymbol)]
 --------------------------------------------------------------------------------
 interpSymbols =
-  [ interpSym setEmp   emp  (FAbs 0 $ FFunc (setSort $ FVar 0) boolSort)
-  , interpSym setEmpty emp  (FAbs 0 $ FFunc intSort (setSort $ FVar 0))
-  , interpSym setSng   sng  (FAbs 0 $ FFunc (FVar 0) (setSort $ FVar 0))
-  , interpSym setAdd   add   setAddSort
-  , interpSym setCup   cup   setBopSort
-  , interpSym setCap   cap   setBopSort
-  , interpSym setMem   mem   setMemSort
-  , interpSym setDif   dif   setBopSort
-  , interpSym setSub   sub   setCmpSort
-  , interpSym setCom   com   setCmpSort
+  [
+  -- TODO we'll probably need two versions of these - one for sets and one for maps
```

Comment:
Still a TODO?

Reply:
Once we start converting Maps, there are likely to be some design choices whether to try making Array theory more polymorphic, or simply have a second set of Array operations specialized for Sets. This TODO is a heads-up for this.